demo.env

```diff
@@ -17,16 +17,16 @@ FEDORA_DB=fedora3
 FEDORA_DB_USER=fedora_admin
 
 ### Fedora database password
-FEDORA_DB_PASS=ild_feddb_2018
+FEDORA_DB_PASS=demo_feddb
 
 ### Drupal database name
-DRUPAL_DB=isle_ld
+DRUPAL_DB=isle_demo
 
 ### Drupal database user
-DRUPAL_DB_USER=isle_ld_user
+DRUPAL_DB_USER=isle_demo_user
```

Comment:
When people pull from upstream, isn't this going to overwrite their customized values in these places? Is this juice worth the squeeze?

Reply:
I guess not as I didn't realize that we've painted ourselves into a corner here. That should be just for demo purposes but instead it sounds like it is a hard coded value that can't ever be changed.

Reply:
I'll revert these changes then and retest. @dwk2 I think unfortunately at this point I cannot easily change values like we thought in https://github.com/Islandora-Collaboration-Group/ISLE/issues/217 I'll retest with just the password changes but not the domain name or container changes (which are probably out of the scope of your request in any event.)

Reply:
Thank you for the careful analysis and I'm okay with these changes being reverted.